documentation/src/main/asciidoc/user_guide/server_protocols_hotrod.adoc

```diff
@@ -1163,6 +1163,9 @@ updated or removed server-side, the cached near cache entries are invalidated.
 If a key is requested after it's been invalidated, it'll have to be re-fetched
 from the server.
 
+WARNING: You should not use `maxIdle` expiration with near caches, as the
+behaviour is undefined.
```

Comment:
*Undefined* is convenient, but is there a reason to not say that accesses to the near cache will be ignored for the purposes of expiration?

Reply:
Yes, let me clear that. I also found a compilation failure.